Makefile

```diff
@@ -83,8 +86,9 @@ AUTHORINO_BRANCH = v$(AUTHORINO_VERSION)
 AUTHORINO_IMAGE_TAG = v$(AUTHORINO_VERSION)
 endif
 
-AUTHORINO_IMAGE_FILE ?= authorino_image
-DEFAULT_AUTHORINO_IMAGE ?= $(shell cat $(AUTHORINO_IMAGE_FILE) || echo $(DEFAULT_REGISTRY)/$(DEFAULT_ORG)/authorino:latest)
+# Build file used to store replaces/authorinoImage options.
+BUILD_CONFIG_FILE ?= build.yaml
+DEFAULT_AUTHORINO_IMAGE ?= $(shell yq e -e '.config.authorinoImage' $(BUILD_CONFIG_FILE) || echo $(DEFAULT_REGISTRY)/$(DEFAULT_ORG)/authorino:latest)
```

Comment:
I guess this will fail if you don't have `yq` in your `PATH` (even if the build config file exists. Maybe `$(YQ)` will do the job?

Reply:
Yep I set $(YQ) here but I think this means any run of the makefile is now a dependency on $(YQ)